modin/pandas/test/test_io.py

```diff
@@ -1396,6 +1396,17 @@ def test_read_parquet_without_metadata(self):
         finally:
             teardown_test_files([parquet_fname, csv_fname])
 
+    def test_read_empty_parquet_file(self):
+        test_df = pandas.DataFrame()
+
+        with tempfile.TemporaryDirectory() as directory:
+            path = f"{directory}/data"
+            os.makedirs(path)
+            test_df.to_parquet(path + "/part-00000.parquet")
+            read_df = pd.read_parquet(path)
+
+            df_equals(test_df, read_df)
```

Comment:
Should we check that its equal with `pandas.read_parquet` as well?

Reply:
Now that you mention it, I think that's the only thing we should check. I switched the equality check to an `eval_io` call which will do that.